scripts/all-tests.sh

```diff
@@ -191,6 +196,7 @@ END
                 # May be unnecessary if we changed the "Skip if there are no test files"
                 # option to True in the jenkins job definitions.
                 mkdir -p reports
+                mkdir -p reports/bok_choy
```

Comment:
This is the change that we need people to have to get the extra shard to pass?
What if we make this a separate commit that we could then cherry-pick?